src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1124,14 +1117,13 @@ namespace Akka.Actor
     }
     public class LocalActorRef : Akka.Actor.ActorRefWithCell, Akka.Actor.IActorRefScope, Akka.Actor.ILocalRef
     {
-        public LocalActorRef(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, System.Func<Akka.Dispatch.Mailbox> createMailbox, Akka.Actor.IInternalActorRef supervisor, Akka.Actor.ActorPath path) { }
-        protected internal LocalActorRef(Akka.Actor.ActorSystem system, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, System.Func<Akka.Dispatch.Mailbox> createMailbox, Akka.Actor.IInternalActorRef supervisor, Akka.Actor.ActorPath path, System.Func<Akka.Actor.LocalActorRef, Akka.Actor.ActorCell> createActorCell) { }
+        public LocalActorRef(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, Akka.Dispatch.MailboxType mailboxType, Akka.Actor.IInternalActorRef supervisor, Akka.Actor.ActorPath path) { }
```

Comment:
There is one and only one constructor for `LocalActorRef`. The `protected internal` one has been removed.